docs/Worksheet-Markdown.md

```diff
@@ -210,7 +210,15 @@ To reference a set of bundles by a search criteria:
 
     % search <keyword-1> ... <keyword-n>
 
-See documentation for `cl search` for more information.
+See [CLI documentation](https://codalab-worksheets.readthedocs.io/en/latest/CLI-Reference/) for `cl search` for more information.
```

Comment:
"for more information about `cl search`"

Reply:
Maybe should be 'See CLI documentation about 'cl search' for more information'?

Reply:
Sure